website/docs/registries/overview.md

```diff
@@ -0,0 +1,7 @@
+# Overview
+
+Scarb is not only a Cairo build toolchain but also a package manager, so it needs support for package storage. Currently, Git is the default for hosting community packages, but their visibility and ease of discovery are greatly limited. That's what registries are for, to have a concentration of packages in one place that are easily discoverable.
```

Comment:
Documentation should focus not on what is changing, but on what is state of the art **now**. Saying that Git is currently the default is unnecessary. At the very moment of publishing these docs, registries will be the default, while Git be the past.

It's a different thing that there is no official registry, but this is a temporary caveat, which should be said as a banner or something in my opinion.

Reply:
Addressed now that the registry is available 😄